docs/source/miscellaneous_environment_variables.md

```diff
@@ -0,0 +1,15 @@
+(miscellaneous_environment_variables)=
+
+# Miscellaneous Environment Variables
+
+```{eval-rst}
```

Comment:
list-table is a supported directive: https://myst-parser.readthedocs.io/en/latest/syntax/tables.html#list-tables - the build is [failing](https://github.com/pytorch/pytorch/actions/runs/15606497502/job/43997480108?pr=155789#step:11:830) because of this.